course/models.py

```diff
@@ -474,6 +474,14 @@ def clean(self):
     def is_valid_language(self, lang):
         return lang == "" or lang in [key for key,name in settings.LANGUAGES]
 
+    @property
+    def default_language(self):
+        language = self.language
+        language_list = [x for x in language.split("|") if x]
+        if language_list:
+            return language_list[0]
+        return "en"
```

Comment:
I think `settings.LANGUAGE_CODE.split('-', 1)[0]` might be better last fallback.